### PR TITLE
Boot Performance Improvements

### PR DESCRIPTION
**Pull request by cperciva**
My formatting is probably terrible, but hopefully the content here is clear enough.

### CODE REVIEW

File: 2021q3/boot-performance.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+link:https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html[OS boot time comparison]
+
+Contact: Colin Percival <cperciva@FreeBSD.org>
+
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is using an EC2 c5.xlarge instance as a reference
+platform and is measuring the time between when the virtual machine enters the
+EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017 and led to a conference presentation, "Profiling
```
Comment (pauamma) [inline 2021q3/boot-performance.adoc:14]: s/2017 and led/2017, leading to/ (flow: avoid "and" repetition)

File: 2021q3/boot-performance.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+the FreeBSD kernel boot", and quickly yielded roughly 4850 ms of improvements
+(starting from a baseline of about 30 seconds).
+
+Since June, another roughly 9790 ms of time has been shaved off the boot
```
Comment (pauamma) [inline 2021q3/boot-performance.adoc:18]: Instinct tells me s/has been/have been/ unless maybe that improvement came from a single change, in which case it comes to taste and dialect differences.

File: 2021q3/boot-performance.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+process, taking it down to approximately 15 seconds.  There is still more work
+to be done; in particular, while the loader and kernel have been profiled, the
+TSLOG system Colin is using does not currently support userland profiling.
+
+Issues are listed on the wiki page as they are identified; the wiki page also
+has instructions for performing profiling.  Users are encouraged to profile
+the boot process on their own systems, in case they experience delays which
+don't show up on the system Colin is using for testing.
+
+This work is being supported by the Colin's FreeBSD/EC2 Patreon.
```
Comment (pauamma) [inline 2021q3/boot-performance.adoc:28]: s/by the /by/

I'd also s/being supported/supported/, but again, taste and dialect differences.

File: 2021q3/boot-performance.adoc (hunk 1 above)
Comment (cperciva) [reply 2021q3/boot-performance.adoc:14]: Thanks!

File: 2021q3/boot-performance.adoc (hunk 2 above)
Comment (cperciva) [reply 2021q3/boot-performance.adoc:18]: I think "9790 ms have been shaved off" but "9790 of time has been shaved off" -- time is singular even if milliseconds are plural.

File: 2021q3/boot-performance.adoc (hunk 3 above)
Comment (cperciva) [reply 2021q3/boot-performance.adoc]: Thanks!

**Review by pauamma — APPROVED** [review]
LGTM.

**Pull request merged**